web-registry/src/pages/CreditClassDetailsSimple.tsx

```diff
@@ -0,0 +1,313 @@
+    padding: theme.spacing(11, 5),
+    overflow: 'auto',
+  },
+  link: {
+    color: theme.palette.secondary.main,
+    fontWeight: 700,
+  },
+  description: {
+    [theme.breakpoints.up('sm')]: {
+      fontSize: theme.typography.pxToRem(22),
+    },
+    [theme.breakpoints.down('sm')]: {
+      fontSize: theme.typography.pxToRem(18),
+    },
+  },
+  marginBottom: {
+    [theme.breakpoints.up('sm')]: {
+      marginBottom: theme.spacing(12),
+    },
+    [theme.breakpoints.down('sm')]: {
+      marginBottom: theme.spacing(8.75),
+    },
+  },
+  sidebarItemMargin: {
+    [theme.breakpoints.up('sm')]: {
+      marginBottom: theme.spacing(8.75),
+    },
+    [theme.breakpoints.down('sm')]: {
+      marginBottom: theme.spacing(8),
+    },
+    '&:last-child': {
+      marginBottom: 0,
+    },
+  },
+  arrow: {
+    fontSize: theme.typography.pxToRem(16),
+  },
+}));
+
+const CreditClassDetailsSimple: React.FC<CreditDetailsProps> = ({
+  dbClass,
+  ledgerClass,
+  metadata,
+}) => {
+  const styles = useStyles();
+
+  const getAllProjects = (): JSX.Element => {
+    const projects = dbClass?.projectsByCreditClassId?.nodes;
+
+    return projects && projects.length > 0 ? (
+      <div className="topo-background-alternate">
+        <MoreProjectsSection
+          classes={{ root: styles.sectionPadding, title: styles.title }}
+          title="Projects"
+          projects={projects}
+        />
+      </div>
+    ) : (
+      <></>
+    );
+  };
+
+  const ApprovedMethodologies: React.FC<{ methodologies: any[] }> = ({
+    methodologies,
+  }): JSX.Element => {
+    const count = methodologies?.length;
+    const firstMethodology = methodologies?.[0];
+
+    return count > 0 ? (
+      <LineItem
+        label="approved methodologies"
+        data={
+          <Box>
+            <Description
+              className={styles.description}
+              key={firstMethodology?.['schema:name']}
+            >
+              {firstMethodology?.['schema:name']}
+            </Description>
+            {count > 1 && (
+              <Link
+                sx={{ display: 'flex', alignItems: 'center' }}
+                href="TODO"
+                target="_blank"
+              >
+                <Label sx={{ mr: 2 }}>{`+ ${count - 1} more`}</Label>{' '}
+                <SmallArrowIcon className={styles.arrow} />
+              </Link>
+            )}
+          </Box>
+        }
+      />
+    ) : (
+      <></>
+    );
+  };
+
+  interface LineItemProps {
+    label: string;
+    data: string | JSX.Element;
+  }
+
+  const LineItem = ({ label, data }: LineItemProps): JSX.Element => {
+    return !!data ? (
+      <Box
+        sx={{
+          width: { xs: '100%', sm: '50%' },
+          pr: 4,
+        }}
+        className={styles.marginBottom}
+      >
+        <Label className={styles.label}>{label}</Label>
+        <Description className={styles.description}>{data}</Description>
+      </Box>
+    ) : (
+      <></>
```

Comment:
Are you returning a fragment here because of typescript complaining if you return `null`?

Both work and there's not much of a perf difference, but as an FYI this is part of why I prefer `React.FC` instead of manually typed functions like this (typescript doesn't complain when you return `null`). 

It seems most are recommending away from `React.FC` because of the implicit types it adds (it'll always add a `children` type, so TS won't error if you try to use children in a place it won't be rendered), but IMO those are really minor problems and it helps avoid writing boilerplate (for `children` type when you use it, and for the return type)

I also prefer the [return early](https://forum.freecodecamp.org/t/the-return-early-pattern-explained-with-javascript-examples/19364) pattern, as it avoids deep nesting... something like 
```tsx
if (!data) return null

return (
  <Box> {...} </Box>
)
```

Reply:
yeah that's better, I must have gotten lazy here. Thanks.